tests/utils/test_utils.go

```diff
@@ -26,6 +28,41 @@ import (
 	clientTests "github.com/jfrog/jfrog-client-go/utils/tests"
 )
 
+type Vulnerability struct {
+	BomRef string `json:"bom-ref"`
+	Id     string `json:"id"`
+}
+
+type EnrichJson struct {
+	Vulnerability []struct {
+		BomRef string `json:"bom-ref,"`
+		Id     string `json:"id"`
+	} `json:"vulnerabilities"`
+}
```

Comment:
```suggestion
type EnrichBom struct {
	Vulnerabilities []Vulnerability `json:"vulnerabilities xml:"Vulnerabilities"`
}
```
Use the struc already defined. you can combine and define xml and json tags at the same object...
Also move it to format

Reply:
I know I can combine but XML is a bit different in structure than JSON, must separate.

Reply:
moved to format